sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -546,6 +569,16 @@ def _close_handler(self):
         self._message_iter = None
         super(ServiceBusReceiver, self)._close_handler()
 
+    def _keep_receiving(self):
```

Comment:
Isn't this essentially just the same as setting the existing `keep_alive_interval` setting to 5?
Why did we need to replace the keep-alive function?